website/docs/r/is_instance_group_membership.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "ibm"
+page_title: "IBM : is_instance_group_membership"
+description: |-
+  Manages Instance Group Membership
+subcategory: "Virtual Private Cloud API"
```

Comment:
The subcategory should be "VPC infrastructure" in all docs and keep it at top before layout